routes/overview_page.py

```diff
@@ -0,0 +1,11 @@
+from routes.utils import file_exists, load_yml_from_file
+
+
+import config
+
+
+def get_services_links():
+    if file_exists(config.SERVICES_LINKS_PATH):
```

Comment:
What about something like this:
```python
if config.SERVICES_LINKS_PATH.is_file():
    return yaml.safe_load(config.SERVICES_LINKS_PATH.read_text())
return yaml.safe_load(config.SERVICES_LINKS_EXAMPLE_PATH.read_text())
```
This might be too much but because you use `pathlib.Path` you don't need `os.path` and the helper function and the result is still readable.

Reply:
😍 that makes sense